include/engine/datafacade/contiguous_internalmem_datafacade.hpp

```diff
@@ -154,7 +155,7 @@ class ContiguousInternalMemoryAlgorithmDataFacade<algorithm::CoreCH>
     : public datafacade::AlgorithmDataFacade<algorithm::CoreCH>
 {
   private:
-    util::ShM<bool, true>::vector m_is_core_node;
+    util::ShM<unsigned, osrm::storage::MemorySetting::SharedMemory>::vector m_is_core_node;
```

Comment:
@duizendnegen Note that the `util::ShM<bool>::vector` class has a special implementation:

https://github.com/Project-OSRM/osrm-backend/blob/master/include/util/shared_memory_vector_wrapper.hpp#L133

where it packs bool values inside individual bits in a large `char` buffer.  This is significantly more compact (probably around 32x) than using `unsigned`.